src/mono/wasm/debugger/DebuggerTestSuite/Tests.cs

```diff
@@ -1523,6 +1523,38 @@ async Task<Result> _invoke_getter(string obj_id, string property_name, bool expe
             }
         }
 
+        [Fact]
```

Comment:
Maybe also add a test for removing the breakpoint again, once it has already been removed?

Reply:
And maybe a test for 1. set, hit, remove, run-don't-hit, set again, hit ?

Reply:
These don't need to block the PR, if we want to get it in today.